product_sticker/views/product_sticker_views.xml

```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="action_product_sticker" model="ir.actions.act_window">
+        <field name="name">Product Stickers</field>
+        <field name="res_model">product.sticker</field>
+        <field name="view_mode">kanban,tree,form</field>
+    </record>
```

Comment:
I miss a menu that calls this action, to be able to review this module independently.

I know this is just a base module, but since it's defining some views, it's useful to expose them somewhere. The common practice is to put it under _Settings > Technical_. After all, with the permissions exposed in this base module, you need access to settings to be able to manage stickers. And that menu is hidden by default without debug mode.